### ISSUE TITLE
Do not pre-style sortable items

### ISSUE THREAD

**Issue by HadrienPierart** [a5hik/ng-sortable#201]
I just got upgraded through a minor upgrade with a bower install and I realized that now, a few styles are automatically added on sortable elements.
As these styles contains margins, it completely broke my design and component...

Please **do not insert default styles** like that **in a library**, especially not without mentionning it in the documentation and through a minor upgrade !

Is there any reason this CSS was added to the library ?

The main culprit seems to be : 
    .as-sortable-item, .as-sortable-placeholder {
      min-height: 20px;
      margin: 1px 2px 5px 1px;
      border-radius: 10px;
    }

Thanks !

**Comment by tobiasbueschel** [comment]
@HadrienPierart good point :) This has already been fixed by the following: https://github.com/a5hik/ng-sortable/commit/5652dbdbd7f831c766762d967b79d63eabb07d5c

@a5hik I think we can close this?
